### ISSUE TITLE
Configure SERVER_PASSWORD has no effect. in 0.19.0

### ISSUE THREAD

**Issue by possible318** [thijsvanloef/palworld-server-docker#225]
## Describe the bug

Configure SERVER_PASSWORD has no effect. in 0.19.0

## To Reproduce

1. Configure SERVER_PASSWORD="xxx"  to  .env  file
2. restart servers

## Expected behavior

Accessing the server with a password

## Logs
```
palworld-server | *****CHECKING FOR EXISTING CONFIG*****
palworld-server | SERVER_NAME="pal server"
palworld-server | SERVER_DESCRIPTION="pal server"
palworld-server | SERVER_PASSWORD="xxx"
palworld-server | ADMIN_PASSWORD="xxx"
palworld-server | PLAYERS=20
palworld-server | PUBLIC_PORT=8211
palworld-server | DIFFICULTY=None
palworld-server | DAYTIME_SPEEDRATE=1.000000
palworld-server | NIGHTTIME_SPEEDRATE=1.000000
palworld-server | EXP_RATE=1.000000
palworld-server | PAL_CAPTURE_RATE=1.000000
palworld-server | PAL_SPAWN_NUM_RATE=1.000000
palworld-server | PAL_DAMAGE_RATE_ATTACK=1.000000
palworld-server | PAL_DAMAGE_RATE_DEFENSE=1.000000
palworld-server | PLAYER_DAMAGE_RATE_ATTACK=1.000000
palworld-server | *****STARTING SERVER*****
palworld-server | ./PalServer.sh -port=8211 -queryport=27015
palworld-server | time="2024-01-30T11:43:42Z" level=info msg="read crontab: /home/steam/server/crontab"
palworld-server | [S_API] SteamAPI_Init(): Loaded local 'steamclient.so' OK.
palworld-server | Shutdown handler: initalize.
palworld-server | Increasing per-process limit of core file size to infinity.
palworld-server | - Existing per-process limit (soft=18446744073709551615, hard=18446744073709551615) is enough for us (need only 18446744073709551615)
palworld-server | CAppInfoCacheReadFromDiskThread took 4 milliseconds to initialize
palworld-server | Setting breakpad minidump AppID = 2394010
palworld-server | [S_API FAIL] Tried to access Steam interface SteamUser021 before SteamAPI_Init succeeded.
palworld-server | [S_API FAIL] Tried to access Steam interface SteamFriends017 before SteamAPI_Init succeeded.
palworld-server | [S_API FAIL] Tried to access Steam interface STEAMAPPS_INTERFACE_VERSION008 before SteamAPI_Init succeeded.
palworld-server | [S_API FAIL] Tried to access Steam interface SteamNetworkingUtils004 before SteamAPI_Init succeeded.
```

## docker-compose.yml contents

```yaml
services:
  palworld:
    image: thijsvanloef/palworld-server-docker:latest
    restart: unless-stopped
    container_name: palworld-server
    stop_grace_period: 30s # Set to however long you are willing to wait for the container to gracefully stop
    ports:
      - 8211:8211/udp
      - 27015:27015/udp # Required if you want your server to show up in the community servers tab
    env_file:
      - .env
    volumes:
      - ./palworld:/palworld/
```

## Additional context

start.sh just writes the configuration to PalWorldSettings.ini via the sed method, without adding the command

> [palworld-server-docker/scripts/start.sh](https://github.com/thijsvanloef/palworld-server-docker/blob/229d29a5787b44e2973030ceaee407472c981757/scripts/start.sh#L51-L55)

**Comment by PyRowMan** [comment]
I reproduce this one :/

**Comment by PyRowMan** [comment]
I can confirm it works well with v0.18

**Comment by lerndmina** [comment]
Can confirm this happens with me too.

**Comment by burkewm** [comment]
> Problem should be resolved in update 0.19.1 thanks to beomseok-park, please pull the latest image and check if your issue has been resolved. If you are still having trouble after updating, please remove your PalWorldSettings.ini and restart your container.

After adding back in my world settings the password requirement also goes away still. without the world config it's fine

**Comment by thijsvanloef** [comment]
Closing this issue to merge all related issues into one: https://github.com/thijsvanloef/palworld-server-docker/issues/239
